assets/javascript/game.js

```diff
@@ -4,42 +4,61 @@ var guessWrong = 0;
 var maxGuesses = 7;
 var winCount = 0;
 var lossCount = 0;
-var wordBank = ["Algebra", "Poodle", "Citizen", "Pizzaz", "Cooperate", "Munchkin", "Flower", "Supercalifragilisticexpialidocious"];
+var wordBank = ['Algebra', 'Poodle', 'Citizen', 'Pizzaz', 'Cooperate', 'Munchkin', 'Flower', 'Supercalifragilisticexpialidocious'];
 
-//Pull from wordBank to give puzzle word
+// Pull from wordBank to give puzzle word, and make sure all buttons are enabled
+// Reset game state to initial state
+// Can add an event listener to a reset button to use this method to reset the game.
 function initializeGame() {
     var gameWord = wordBank[Math.floor(Math.random() * wordBank.length)];
     var answerContainer = document.getElementById('answer');
-    answerContainer.innerHTML = "";
+    answerContainer.innerHTML = '';
+
     for (var i = 0; i < gameWord.length; i++) {
         var letter = gameWord[i];
-        var upperLetter = letter.toUpperCase(); 
-        answerLetters[upperLetter] = true;  //making sure all cases of letter work
-        var letterDiv = document.createElement('div');
-        letterDiv.classList.add(['letter-container'])
-        letterDiv.classList.add(['p-3'])
-        var letterSpan = document.createElement('span');
-        letterSpan.setAttribute('name', upperLetter);
-        letterSpan.innerHTML = letter;
-        letterSpan.classList.add(['hide-letter']);  //using class to hide display of letters
-        var underscoreSpan = document.createElement('span');
-        underscoreSpan.innerHTML = '_'; //creating "_" to display under letters
-        underscoreSpan.classList.add(['under-letter'])
-        letterDiv.appendChild(letterSpan);
-        letterDiv.appendChild(underscoreSpan);
-        answerContainer.appendChild(letterDiv);
+        var upperLetter = letter.toUpperCase();
+        answerLetters[upperLetter] = true; //making sure all cases of letter work
+        answerContainer.appendChild(createAnswerDiv(gameWord, letter, upperLetter));
```

Comment:
here is where I'm calling the function to create the answer div and passing it any arguments that it needs.
so before we were doing
```
var div = document.createElement('div');
/// do stuff to the div
answerContainer.appendChild(div);
```

what I did was write the function `createAnswerDiv` which will create the div, do some stuff and return the created div.
this makes the loop more readable.